bin/writer.rpi3

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019 Google LLC. All rights reserved.
```

Comment:
I'm a little hesitant to introduce "writer" as a new "application" or "hook" name; it's rather generic.

If we consider this as just another method of flashing, perhaps "flash.rpi_mounted_disk" would be better?

If we want a new "hook" or "application" name, perhaps something a bit more descriptive like "writer.rpi_mounted_disk" would be better; "rpi3" isn't descriptive enough since I have an implementation (which unfortunately I never pushed to this repo) that uses WiFi SD cards instead, which demonstrates that there are multiple methods for writing files to an RPi.

Reply:
My thinking is that several boards will need to write to a mounted disk, but each has its own files to write. We don't want the mount logic to be duplicated everywhere. So I think we need a new method.

Will rename it.

(Yes would be good to push your WiFi SD thing as I have a few of those)